Warmup-1/front22/test/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import { assert } from "chai";
+import { front22 } from "../src";
+// Given a string, take the first 2 chars and return the string with the 2 chars added at both the front and back, so "kitten" yields"kikittenki". If the string length is less than 2, use whatever chars are there.
```

Comment:
Please add a test for when the string is less than 2 characters long

Reply:
Sure, test added.